cypress/pages/wizard.po.ts

```diff
@@ -82,6 +86,17 @@ export class WizardPage {
     return cy.get('#km-node-count-input');
   }
 
+  static getAutocompleteOption(value: string): Cypress.Chainable {
```

Comment:
This might be dangerous to use on i.e. different provider pages. It will pick up the first `mat-option` on the page and it might be easily missed during debugging. It will not be possible to actually pick wanted autocomplete field.